Update README.md

AnushkaSingh91 opened the pull request:
Fixes: #9 
Updated Readme.md and added the required documentations.

AnushkaSingh91 committed:
Update README.md

Updated Readme.md and added the required documentations.

VighneshManjrekar commented on README.md:
```diff
@@ -8,3 +8,51 @@ https://kudacam.netlify.app/
+Or if you find an existing issue in the [issue](https://github.com/CSI-CATT-DMCE/kudamcam/issues) section in the repository that has not been assigned previously, then ask the maintainer to assign that issue so that you can start working on it.
+
+
+## How to create a Pull Request?
```
Need more detailed steps.

VighneshManjrekar commented on README.md:
```diff
@@ -8,3 +8,51 @@ https://kudacam.netlify.app/
+## KickStart to Open Source Event 
+
+## What is Hacktoberfest?
+A month-long celebration from October 1st - 31st sponsored by [Digital Ocean](https://hacktoberfest.digitalocean.com/) and [GitHub](https://github.com/blog/2433-celebrate-open-source-this-october-with-hacktoberfest) to get people involved in [Open Source](https://github.com/open-source). Create your very first pull request and contribute to the open source developer community.
```
Create your very first pull request and contribute to the open source developer community.
 Remove this line

AnushkaSingh91 committed:
Update Readme.md

Made changes in the readme by making the issue content detailed along with deleting the required lines .

AnushkaSingh91 commented:
@VighneshManjrekar Kindly check the pr now.

VighneshManjrekar commented on README.md:
```diff
@@ -8,3 +8,70 @@ https://kudacam.netlify.app/
+ 4. Lastly click on comment and wait for the issue to be assigned by the maintainer.
+ 
+ 
+ Once the issue is assigned
```
Add the below content in How to create a pull request section
replace your content with below text accordingly:
[Fork](https://github.com/CSI-CATT-DMCE/kudamcam/fork) the repo
After cloning make sure you create a new branch by using <br> `git checkout -b my-branch`.

VighneshManjrekar reviewed (changes requested):
Add the below content in How to create a pull request section
replace your content with below text accordingly:
[Fork](https://github.com/CSI-CATT-DMCE/kudamcam/fork) the repo
After cloning make sure you create a new branch by using
`git checkout -b my-branch`

AnushkaSingh91 committed:
Update Readme.md

Made changes in the Readme.md

AnushkaSingh91 commented:
@VighneshManjrekar Kindly check the PR.

VighneshManjrekar committed:
Update README.md
